themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
I've implemented an example for this. There's still one problem. Items is an array of fields. The 2nd key won't always be the language. So we need a mechanism to configure an iteration. 

I'll ask my Ebsco contact if RecordInfo *always* contains the same Info as Items. If so, we might just drop Items support as it makes EDS really hard to read.